test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js

```diff
@@ -634,6 +634,33 @@ describe( `[${ host }] Calypso Gutenberg Site Editor Tracking: (${ screenSize })
 			);
 		} );
 
+		describe( 'tracks template part creation and replacement', function () {
+			it( 'Tracks "wpcom_block_editor_create_template_part', async function () {
+				const editor = await SiteEditorComponent.Expect( this.driver );
+				await editor.addBlock( 'Header', 'template-part\\/header', 'Block: Template Part' );
+				const createNewHeaderLocator = driverHelper.createTextLocator(
+					By.css( '.wp-block-template-part:last-of-type .components-placeholder button' ),
+					'New header'
+				);
+				await driverHelper.clickWhenClickable( this.driver, createNewHeaderLocator );
```

Comment:
Im not sure why this isnt working.  I can verify the css selector targets the buttons in the placeholder.  And I can verify the button's text is 'New header' ....  but nope.